modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/IS/analytics/config/identity_event_analytics_enabled.toml

```diff
@@ -0,0 +1,39 @@
+[server]
+pattern = "DEV"  #possible values, DEV,HA,HA_WITH_ANALYTICS
+hostname = "localhost"
+node_ip = "127.0.0.1"
+base_path = "https://localhost:9853"
```

Comment:
This a `url` not a `path`. So the key has to be something like `base`, `host`

Reply:
"server.base_path" is the configuration that use to identify the server url and it is only referenced in the test tomls. We cannot change this configuration as this is used across the multiple products.

Reply:
Ack.